app/services/force/listing_service.rb

```diff
@@ -211,5 +218,36 @@ def self.array_sort!(listing)
       listing['Units'] = stub_unit_data(listing['Units'])
       listing
     end
+
+    # TODO: Remove this method when we no longer need to stub data.
+    private_class_method def self.stub_rental_listing_data(listing)
```

Comment:
hm, this tenure info should be available via the API now 🤔

Reply:
if it's available then let's add it here